src/main/scala/mesosphere/marathon/core/launchqueue/impl/AppTaskLauncherActor.scala

```diff
@@ -205,33 +206,30 @@ private class AppTaskLauncherActor(
   }
 
   private[this] def receiveTaskLaunchNotification: Receive = {
-    case TaskLaunchSourceDelegate.TaskLaunchRejected(taskInfo, reason) if inFlight(taskInfo) =>
-      removeTask(taskInfo.getTaskId)
+    case TaskOpSourceDelegate.TaskLaunchRejected(op, reason) if inFlight(op) =>
+      removeTask(op.taskId)
       tasksToLaunch += 1
-      log.info(
-        "Task launch for '{}' was REJECTED, reason '{}', rescheduling. {}",
-        taskInfo.getTaskId.getValue, reason, status)
+      log.info("Task launch for '{}' was REJECTED, reason '{}', rescheduling. {}", op.taskId, reason, status)
       OfferMatcherRegistration.manageOfferMatcherStatus()
 
-    case TaskLaunchSourceDelegate.TaskLaunchRejected(
-      taskInfo, AppTaskLauncherActor.TASK_LAUNCH_REJECTED_TIMEOUT_REASON
+    case TaskOpSourceDelegate.TaskLaunchRejected(
+      op, AppTaskLauncherActor.TASK_LAUNCH_REJECTED_TIMEOUT_REASON
       ) =>
 
-      log.debug("Unnecessary timeout message. Ignoring task launch rejected for task id '{}'.",
-        taskInfo.getTaskId.getValue)
+      log.debug("Unnecessary timeout message. Ignoring task launch rejected for task id '{}'.", op.taskId)
```

Comment:
Curiosity: Does the log message say that the log message is unnecessary or how should this be understood? Is this a _should not happen_ case?